### PR TITLE
frontend: add query hydration to resolver

### PR DESCRIPTION
**Pull request by dschaller**
### Description

Add query param hydration support to workflows that utilize the resolver.

state

### Testing Performed

manual

### CODE REVIEW

File: frontend/packages/core/src/Resolver/index.tsx [hunk 1]
```diff
@@ -91,6 +92,13 @@ const Resolver: React.FC<ResolverProps> = ({
     );
   };
 
+  const [searchParams, ,] = useSearchParams();
```
Comment (danielhochman) [inline frontend/packages/core/src/Resolver/index.tsx:95]: same thing right?
```suggestion
  const [searchParams] = useSearchParams();
```

**Review by danielhochman — COMMENTED** [review]
how about setting it on submit of the search resolver?

File: frontend/packages/core/src/Resolver/index.tsx (hunk 1 above)
Comment (dschaller) [reply frontend/packages/core/src/Resolver/index.tsx:95]: ah yupp :)

**Comment by dschaller** [comment]
👍  that should work. We intentionally reset it on restart so there shouldn't be any issues with that

**Review by danielhochman — COMMENTED** [review]
last request, can we populate the search field with defaultValue as q in case submission fails and its needs to be edited?

**Review by danielhochman — APPROVED** [review]
great! 26 lines buys us at least a month or two on real state hydration

**Comment by dschaller** [comment]
just have to remember to clean it up ;)

**Pull request merged**